src/main/java/carpet/script/utils/SystemInfo.java

```diff
@@ -133,7 +133,7 @@
         put("java_system_cpu_load", c -> {
             OperatingSystemMXBean osBean = ManagementFactory.getPlatformMXBean(
                     OperatingSystemMXBean.class);
-            return new NumericValue(osBean.getSystemCpuLoad());
+            return new NumericValue(osBean.getCpuLoad());
```

Comment:
What changed here?

Reply:
The old one is deprecated because of "being a historically named method". The old one calls the new one so they do the same.